doc/src/build/move.md

```diff
@@ -374,7 +374,7 @@ Since we are developing a fantasy game, in addition to the mandatory
 `Coin` struct), our asset has both `magic` and `strength` fields
 describing its respective attribute values. Please note that we need
 to import the
-[ID package](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/sources/id.move) from
+[Object package](https://github.com/MystenLabs/sui/blob/main/crates/sui-framework/sources/object.move) from
```

Comment:
I really need to go through all these docs. The wording here is sort of misleading about what is going on Move.

I plan on going through all these Move related docs once the framework changes finalize (hopefully later this week)

Reply:
Yes, please! Thanks, Todd.